client/src/components/ObjectStore/ObjectStoreRestrictionSpan.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <span class="stored-how" v-b-tooltip.hover :title="title">{{ text }}</span>
```

Comment:
This is just to fix the `eslint` warning about the [ordering of attributes](https://eslint.vuejs.org/rules/attributes-order.html).
```suggestion
    <span v-b-tooltip.hover class="stored-how" :title="title">{{ text }}</span>
```